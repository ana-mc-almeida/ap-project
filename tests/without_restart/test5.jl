
include("./common.jl")

handling(DivisionByZero =>
    (c) -> println("I saw it too")) do
    handling(DivisionByZero =>
        (c) -> println("I saw a division by zero")) do
        print(reciprocal(0))
    end
end