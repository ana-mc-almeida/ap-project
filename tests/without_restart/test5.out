I saw a division by zero
I saw it too
ERROR: DivisionByZero()